cpp/src/rolling/legacy/rolling.cu

```diff
@@ -41,6 +41,9 @@
 #include <types.h.jit>
 #include <types.hpp.jit>
 
+#include <cudf/scalar/scalar.hpp>
```

Comment:
This is a libcudf++ include, not a legacy include. Also, you don't seem to use anything in this header.
```suggestion
```

Reply:
Whoops.  Good catch.  By chance, that happened to get me what I needed (rmm::device_scalar) but the include should have been rmm/device_scalar.hpp